rest/httpx/responses.go

```diff
@@ -172,6 +173,26 @@ func doHandleError(w http.ResponseWriter, err error, handler func(error) (int, a
 	}
 }
 
+func doMarshalJson(v any) ([]byte, error) {
```

Comment:
It's a non-exported method, and not used in package scope. What's the purpose of this method?